R/methods.R

```diff
@@ -40,6 +29,22 @@ format.lint <- function(x, ..., width = getOption("lintr.format_width")) {
   )
 }
 
+build_line_ref <- function(x) {
+  line_ref <- paste0(
+    x$filename, ":",
+    as.character(x$line_number), ":",
+    as.character(x$column_number)
+  )
+
+  create_clickable_link <- cli::ansi_has_hyperlink_support()
```

Comment:
```suggestion
  create_clickable_link <- cli::ansi_has_hyperlink_support() && !grepl("\\{|\\}", x$filename)
```

@AshesITR, you are right, it would error, but I have never seen a file name with `{` in them. Could add this check to prevent this.